### ISSUE TITLE
Delete possibly dangerous text files afterwards

### ISSUE THREAD

**Issue by kartagis** [drupal-composer/drupal-composer-project#470]
Text files such as CHANGELOG.txt could easily be exploited on a Drupal website. These text files should be removed post-install.

**Comment by normanlolx** [comment]
Nice to see how easily you customized the ScriptHandler to fit your needs. Nevertheless I'd classify this as an opinion-based issue.

This template is a kickstart kit to get your Drupal up and running. Those text files may get removed to reach a certain level of security through obscurity, sure. But this probably should be handled manually according every maintainer's own needs.

**Comment by kartagis** [comment]
Many devs might not know this is a good practice and may fall into the trap.

**Comment by shrop** [comment]
We used to remove CHANGELOG.txt in the Guardr Drupal security distribution. This issue has great commentary on pros and cons of deleting the files: https://www.drupal.org/project/guardr/issues/1878172. The larger argument is that it is security by obscurity.

We found the big issue to be keeping up with the patch to remove the CHANGELOG.txt. We now recommend that users block any files they want using Apache / Nginx rules. Some security teams will require this and some realize that it doesn't really prevent a site from being fingerprinted.
